Bug: test_get_risk fails — max_positions default changed from 15 to 50

## Problem

`tests/unit/test_clubhouse.py::TestDataLayer::test_get_risk` fails on main:

```
assert risk.max_positions == 15
E       assert 50 == 15
```

The test expects `max_positions=15` but the current default is `50`. Looks like the default was updated without adjusting the test assertion.

## Steps to reproduce

```bash
uv run pytest tests/unit/test_clubhouse.py::TestDataLayer::test_get_risk -x
```

## Expected

Test should pass on main.

--- a/tests/unit/test_clubhouse.py
+++ b/tests/unit/test_clubhouse.py
@@ -35,10 +35,12 @@ from gimmes.store.migrations import run_migrations
 
 
 @pytest.fixture(autouse=True)
-def _reset_config_cache(monkeypatch):
-    """Reset the clubhouse data module's config cache and ensure driving_range mode."""
+def _reset_config_cache(monkeypatch, tmp_path):
+    """Reset the clubhouse data module's config cache and use isolated config."""
     import gimmes.clubhouse.data as data_mod
+    import gimmes.config as config_mod
     monkeypatch.setenv("GIMMES_MODE", "driving_range")
+    monkeypatch.setattr(config_mod, "GIMMES_HOME", tmp_path)
     data_mod._cached_config = None
     data_mod._kalshi_client = None
     yield

